src/app/components/MediaLoader/utils/buildSettings.ts

```diff
@@ -45,6 +54,9 @@ const buildSettings = ({
     },
     ...(counterName && { counterName }),
     ...(isTestRequested() && { mediator: { host: 'open.test.bbc.co.uk' } }),
+    ...(embedded && {
+      externalEmbedUrl: embedUrl,
+    }),
```

Comment:
Is it worth just setting `externalEmbedUrl` for every use case, rather than just for `embedded`? I don't think it should affect anything having it set even for players that aren't being embedded. SMP should just ignore. Worth testing at least on a Live page thats using this player.